stubs/docutils/docutils/transforms/__init__.pyi

```diff
@@ -1,14 +1,31 @@
-from _typeshed import Incomplete
+from collections.abc import Iterable
+from types import ModuleType
+from typing import Any
 
-from docutils.nodes import Node, document
+from docutils import ApplicationError, Component, TransformSpec, nodes
+from docutils.nodes import Node
+
+class TransformError(ApplicationError): ...
 
 class Transform:
-    def __init__(self, document: document, startnode: Node | None = None): ...
-    def __getattr__(self, name: str, /) -> Incomplete: ...
+    default_priority: int | None
+    document: nodes.document
+    startnode: Node | None
+    language: ModuleType
```

Comment:
It's more than just a module, it's a module with specific attributes. It's more important to have the protocol rather than the module type IMO.

The protocol is:

```
class _LanguageModule(Protocol):
    labels: dict[str, str]
    author_separators: list[str]
    bibliographic_fields: list[str]
```